Lib/test/test_math.py

```diff
@@ -974,6 +974,40 @@ def __index__(self):
                 with self.assertRaises(TypeError):
                     math.isqrt(value)
 
+     def test_lcm(self):
+        lcm = math.lcm
+        self.assertEqual(lcm(0, 0), 0)
```

Comment:
Most of the right-hand sides are wrong for these tests: it looks as though you still have the `gcd` values here instead of the expected `lcm` values.